script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -547,7 +547,7 @@ But my body won't move.
 My unfettered eyes,
  able to perceive the threadbare nature of this run-down world,
  are drawn as though entranced to the dark void Arcueid vanished into,
- and I gaze fixedly across that threshold of darkness.
+ as I gaze fixedly across that threshold of darkness.
```

Comment:
I think keep `and`, since the temporality is gaze drawn -> he then stares across boundary where the shadows begin, rather than gazing at it _as_ his eyes are still moving to it